README.md

```diff
@@ -1,9 +1,8 @@
 # GitSavvy
 
-[![Build Status](https://travis-ci.org/divmain/GitSavvy.svg?branch=master)](https://travis-ci.org/divmain/GitSavvy)
+[![Build Status](https://travis-ci.org/timbrel/GitSavvy.svg?branch=master)](https://travis-ci.org/timbrel/GitSavvy)
 [![AppVeyor branch](https://img.shields.io/appveyor/ci/divmain/GitSavvy/master.svg)](https://ci.appveyor.com/project/divmain/GitSavvy)
```

Comment:
This still points to `divmain/GitSavvy`. Is this a mistake?

Reply:
Not really, https://ci.appveyor.com/project/divmain/GitSavvy is still functioning (it was because @divmain granted me access to this particular project in Appveyor so I could activate it).

However, as I have stated [here][1], the best resolution is to create an organization account for timbrel and added all of us as administrators.

[1]: https://github.com/timbrel/GitSavvy/pull/1180#issuecomment-546513408